packages/docs/layouts/default.vue

```diff
@@ -31,16 +32,14 @@
 <script setup lang="ts">
 import { useColors } from 'vuestic-ui'
 import { useDocsScroll } from '../composables/useDocsScroll';
+import { useIsMounted } from 'vuestic-ui/src/composables/useIsMounted'
 
-const cookie = useCookie('vuestic-theme')
```

Comment:
How do we set the theme that the user has last selected?
Instead we need to use localStorage. This can be done in this task or you need to open a new one.